assets/prefabs/items/locoItem.prefab

```diff
@@ -0,0 +1,14 @@
+{
+    "parent": "engine:iconItem",
+    "DisplayName": {
+        "name": "Locomotive"
+    },
+    "Item": {
+        "name": "Minecart",
+        "icon": "Rails:minecart#loco",
```

Comment:
If you change this to simply be `Rails:loco` I think it would still work? Then you can delete the atlas file entirely. 

And in fact it should probably be `AdvancedRails:loco` - it might be working here because it is actually still resolving the `loco` atlas entry from Rails, which is itself out of date. Which is another reason I dislike atlas files, we can't easily remove _just_ the logo from the atlas texture in Rails, sigh ... 😁